omod/src/main/java/org/openmrs/module/kenyaemr/web/controller/KenyaemrCoreRestController.java

```diff
@@ -584,11 +584,16 @@ public Object getHeiOutcomeEncounter(@RequestParam("patientUuid") String patient
         SimpleObject heiOutcomeResponseObj = new SimpleObject();
         PatientIdentifierType heiNumber = MetadataUtils.existing(PatientIdentifierType.class, MchMetadata._PatientIdentifierType.HEI_ID_NUMBER);
         Patient patient =  Context.getPatientService().getPatientByUuid(patientUuid);
+        if (patient == null || heiNumber == null) {
```

Comment:
Do we really need this here? The patient will never be null

Reply:
> Do we really need this here? The patient will never be null

Make sense unnecessary check. I have made changes. Thanks